.vscode/settings.json

```diff
@@ -19,5 +19,6 @@
   "[typescript]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode",
     "editor.formatOnSave": true
-  }
+  },
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
This one was added by vscode automatically. I don't mind reverting it, but it seems reasonable. It tells vscode to use the local version.

Reply:
yes, good idea 👍